pdh_json_updater/update_json.py

```diff
@@ -8,6 +8,8 @@
 
 SCRYFALL_SETS_SEARCH_URL = "https://api.scryfall.com/sets?order%3Dreleased"
 ILLEGAL_SET_TYPES = ["alchemy", "token", "memorabilia"]
+# types of sets that would have prerelease events and therefore release early
+PRERELEASE_SET_TYPES = ["core", "expansion", "masters", "funny"]
```

Comment:
So these are the set types I could think of that have prereleases. expansions are normal block sets, commander masters is a masters set, and unfinity is a "funny". "Commander" set code is for precons.
[https://scryfall.com/docs/api/sets](https://scryfall.com/docs/api/sets)